WhirlyGlobeSrc/Android/src/com/mousebird/maply/RendererWrapper.java

```diff
@@ -132,9 +140,50 @@ public void run() {
 		if (valid)
 			maplyRender.doRender();
 
+		if (doScreenshot) {
```

Comment:
The renderLock is really just for shutdown so it's not likely to cause any problems.
But this is the right way to do it.